pkgs/development/python-modules/qbittorrent-api/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, six
+, urllib3
+, packaging
+, setuptools
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "qbittorrent-api";
+  version = "2023.7.52";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-RHOupNo0jteUpxcxAojOfnBGGBt293j0OCHeKEritpQ=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    six
+    urllib3
+    packaging
+  ];
+
+  buildInputs = [
```

```suggestion
  nativeBuildInputs = [
```
I think we don't need these inputs in run-time.

---

Made the change. Sorry, that was an oversight.